scripts/variantstore/docs/aou/AOU_DELIVERABLES.md

```diff
@@ -2,11 +2,11 @@
 
 ## Setup
 - Create a Terra workspace
-  - using your @pmi-ops.org account 
-  - with the `allofus-drc-wgs-dev` Terra Billing account 
-  - in the `AOU_DRC_WGS_OPERATORS` Authorization Domain
-  - Share it with `dsp-variant-team@firecloud.org` so the rest of the team can see it (Reader) and with the individual users who will be running the workflows (Owner).
-- Populate the workspace with the following:
+  - Request a new AoU workspace as described in Section 3.D. of [AoU DRC Protocols](https://docs.google.com/document/d/1ooK0wbHLgSueiepjTTyLLI6Zz7vi1-GhKTjmCd8ZHwU/edit?usp=sharing).
+  - As described in the "Getting Started" of [Operational concerns for running Hail in Terra Cromwell/WDL](https://docs.google.com/document/d/1_OY2rKwZ-qKCDldSZrte4jRIZf4eAw2d7Jd-Asi50KE/edit?usp=sharing), this workspace will need permission in Terra to run Hail dataproc clusters within WDL. Contact Emily to request this access as part of setting up the new workspace.
+- Create and push a Feature Branch (e.g. `EchoCallset`) to the ah_var_store GitHub repo
+    - Update the .dockstore.yml file on that feature branch to add a branch designation for all the WDLs that will be loaded into the workspace in the next step:
+- Once the workspace requested above has been created and permissioned, populate with the following:
```

Comment:
Shouldn't this link the the .ipynb file in our repo instead of the workspace?

Reply:
Updated.